openedx_webhooks/views/jira.py

```diff
@@ -188,18 +188,17 @@ def issue_opened(issue, bugsnag_context=None):
         # goes to, so a transition to go from "Open" to "In Progress" may be
         # named something like "Start Work".
         transitions = {t["name"]: t["id"] for t in transitions_resp.json()["transitions"]}
-        # We attempt to transition the issue into the "Open" state, so we
-        # look for a transition named "Open"
-        if "Open" in transitions:
-            new_status = "Open"
-        # UX uses "Design Backlog" instead of "Open", so check for that instead
-        elif "Design Backlog" in transitions:
-            new_status = "Design Backlog"
-        # If it's an OSPR subtask (used by teams to manage reviews), transition to team backlog
-        elif to_unicode(issue["fields"]["project"]["key"]) == "OSPR" and issue["fields"]["issuetype"]["subtask"]:
-            new_status = "To Backlog"
+        # We attempt to transition the issue into the "Open" state for the given project
+        # (some projects use a different name), so look for a transition with the right name
+        for state_name in ["Open", "Design Backlog", "To Do"]:
+            if state_name in transitions:
+                new_status = state_name
         else:
```

Comment:
Word - yes. I did a bit differently so that we don't check the is_ospr/is_subtask logic if we already have a new status.